libraries/networking/src/EntityScriptClient.cpp

```diff
@@ -15,9 +16,12 @@ GetScriptStatusRequest::~GetScriptStatusRequest() {
 
 void GetScriptStatusRequest::start() {
     auto client = DependencyManager::get<EntityScriptClient>();
-    qDebug() << "Sending script status request";
-    client->getEntityServerScriptStatus(_entityID, [this](bool responseReceived) {
-        qDebug() << "Received script status request";
+    client->getEntityServerScriptStatus(_entityID, [this](bool responseReceived, bool isRunning, EntityScriptStatus status, QString errorInfo) {
+        responseReceived = responseReceived;
+        isRunning = isRunning;
+        status = status;
+        errorInfo = errorInfo;
```

Comment:
I'm pretty sure you're just doing `a = a` here. You need to alter the name of the arguments.